src/views/List.jsx

```diff
@@ -1,15 +1,45 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleTextChange = (event) => {
+		setSearchItem(event.target.value);
+	};
+
+	const filteredItems = data.filter((item) => {
+		return item.name.toLowerCase().includes(searchItem.toLowerCase());
```

Comment:
Nicely done ensuring the search is case-insensitive by converting both the `item.name` and `searchItem` to lowercase!